.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-      - master
+      - '*'
```

Comment:
we chatted about this, but doing this so that we get CI running on all of our branches, not just PR's. it introduces a double run on PR open, but seems worth it. Travis did the same thing, not sure if there's a way around it